src/App.js

```diff
@@ -14,11 +16,94 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    visibility: false,
+    goods: [...goodsFromServer],
+    selected: 1,
+  }
+
+  start = () => {
+    this.setState(
+      {
+        visibility: true,
+      },
+    );
```

Comment:
не переноси скобки объекта, исправь везде
```suggestion
    this.setState({
      visibility: true,
    });
```

Reply:
fixed